paddle/fluid/memory/allocation/allocator_facade_test.cc

```diff
@@ -19,20 +19,17 @@
 #ifdef PADDLE_WITH_CUDA
 DECLARE_double(fraction_of_gpu_memory_to_use);
 DECLARE_double(fraction_of_cuda_pinned_memory_to_use);
+DECLARE_uint64(gpu_init_memory_in_mb);
```

Comment:
Maybe change it to be `initial_gpu_memory_in_mb`?  Because we have another flag named `initial_cpu_memory_in_mb`.